lib/streamlit/elements/streamlit_plotly_theme.py

```diff
@@ -0,0 +1,193 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import plotly.graph_objects as go
+import plotly.io as pio
+
+# This is the streamlit theme for plotly where we pass in a template.data
+# and a template.layout.
+
+# Template.data is for changing specific graph properties in a general aspect
+# such as Contour plots or Waterfall plots.
+
+# Template.layout is for changing things such as the x axis and fonts and other
+# general layout properties for general graphs.
+
+# We pass in temporary colors to the frontend and the frontend will replace
+# those colors because we want to change colors based on the background color.
+
+# Start at #0000001 because developers may be likely to use #000000
+CATEGORY_0 = "#000001"
+CATEGORY_1 = "#000002"
+CATEGORY_2 = "#000003"
+CATEGORY_3 = "#000004"
+CATEGORY_4 = "#000005"
+CATEGORY_5 = "#000006"
+CATEGORY_6 = "#000007"
+CATEGORY_7 = "#000008"
+CATEGORY_8 = "#000009"
+CATEGORY_9 = "#000010"
+
+SEQUENTIAL_10 = "#000011"
+SEQUENTIAL_11 = "#000012"
+SEQUENTIAL_12 = "#000013"
+SEQUENTIAL_13 = "#000014"
+SEQUENTIAL_14 = "#000015"
+SEQUENTIAL_15 = "#000016"
+SEQUENTIAL_16 = "#000017"
+SEQUENTIAL_17 = "#000018"
+SEQUENTIAL_18 = "#000019"
+SEQUENTIAL_19 = "#000020"
+
+DIVERGING_20 = "#000021"
+DIVERGING_21 = "#000022"
+DIVERGING_22 = "#000023"
+DIVERGING_23 = "#000024"
+DIVERGING_24 = "#000025"
+DIVERGING_25 = "#000026"
+DIVERGING_26 = "#000027"
+DIVERGING_27 = "#000028"
+DIVERGING_28 = "#000029"
+DIVERGING_29 = "#000030"
+DIVERGING_30 = "#000031"
+
+INCREASING_31 = "#000032"
+DECREASING_32 = "#000033"
+TOTAL_33 = "#000034"
+
+GRAY_30_34 = "#000035"
+GRAY_70_35 = "#000036"
+GRAY_90_36 = "#000037"
+BG_COLOR_37 = "#000038"
+FADED_TEXT_05_38 = "#000039"
+BG_MIX_39 = "#000040"
+
+BODY_FONT = '"Source Sans Pro", sans-serif'
```

Comment:
nit: is this really necessary, so will the wrong font be picked up if we are not including this in our placeholder theme? Just want to make sure that we are including only the minimal necessary non-color aspects in this theme here.

Reply:
Tested and it doesn't seem to do anything so removed.